javascript/chronometer.js

```diff
@@ -1,26 +1,41 @@
 class Chronometer {
-  constructor() {
-    // ... your code goes here
-  }
-  startClick(callback) {
-    // ... your code goes here
-  }
-  getMinutes() {
-    // ... your code goes here
-  }
-  getSeconds() {
-    // ... your code goes here
-  }
-  twoDigitsNumber() {
-    // ... your code goes here
-  }
-  stopClick() {
-    // ... your code goes here
-  }
-  resetClick() {
-    // ... your code goes here
-  }
-  splitClick() {
-    // ... your code goes here
-  }
-}
+    constructor() {
+        this.currentTime = 0;
+        this.intervalId = 0;
+    }
+    startClick(callback) {
+        this.intervalId = setInterval(() => {
+            this.currentTime += 1;
+            if (callback) callback();
+        }, 1000);
+    }
+
+    getMinutes() {
+        let minutes = Math.floor(this.currentTime / 60)
+        return minutes
+    }
+
+    getSeconds() {
+        if (!this.currentTime)
+            return 0;
+        return this.currentTime - this.getMinutes() * 60;
```

Comment:
You can get a simpler solution by using the [modulo operator](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Math#Arithmetic_operators)
```
return this.currentTime % 60;